kolibri/core/assets/src/views/sync/FacilityAdminCredentialsForm.vue

```diff
@@ -60,9 +60,20 @@
       device: {
         type: Object,
         required: true,
-        validator(val) {
-          return val.name && val.id && val.baseurl;
-        },
+        validator: objectValidator({
```

Comment:
Where is `objectValidator` coming from here?

Reply:
#8878 Hello, I was following the structure mentioned in this PR!

Reply:
I think what @nucleogenesis means is that you will need to import this function from somewhere.

These are the functions available for import, for example:  https://github.com/learningequality/kolibri/blob/39363497bcf8bee78e9faa11a76cb726a6ba6b29/packages/kolibri/styles/internal/initializeTheme.js#L8